techhive/routes/authRoutes.js

```diff
@@ -1,22 +1,31 @@
 import express from 'express';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
 import { User } from '../models/index.js';
 
 const router = express.Router();
 
 // Route for user signup
 router.post('/signup', async (req, res) => {
   try {
-    const { username, email, password, role } = req.body;
```

Comment:
Why are we removing role here? Do we want a user to specify their role (employer vs. applicant) during signup?